runtime/src/accounts_index.rs

```diff
@@ -75,11 +75,8 @@ impl<T: Clone> AccountsIndex<T> {
     ) -> Option<(RwLockReadGuard<SlotList<T>>, usize)> {
         self.account_maps.get(pubkey).and_then(|list| {
             let lock = list.read().unwrap();
-            if let Some(found_index) = self.latest_slot(ancestors, &lock) {
-                Some((lock, found_index))
-            } else {
-                None
-            }
+            self.latest_slot(ancestors, &lock)
```

Comment:
How about:

```rust
let found_index = self.latest_slot(ancestors, &lock)?;
(lock, found_index)
```

Reply:
Thanks!  This doesn't work because it would return the tuple rather than an `Option`.

Reply:
Right, would need to be `Some((lock, found_index))`